### PR TITLE
Updated READMEs for examples: LLM Embedding-Based Named Entity Recognition, nuScenes, Objectron, Open Photogrammetry Format, Raw Mesh

### PR DESCRIPTION
**Pull request by andreasnaoum**
Updated READMEs for the examples:
- LLM Embedding-Based Named Entity Recognition
- nuScenes
- Objectron
- Open Photogrammetry Format
- Raw Mesh

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested the web demo (if applicable):
  * Using newly built examples: [app.rerun.io](https://app.rerun.io/pr/5653/index.html)
  * Using examples from latest `main` build: [app.rerun.io](https://app.rerun.io/pr/5653/index.html?manifest_url=https://app.rerun.io/version/main/examples_manifest.json)
  * Using full set of examples from `nightly` build: [app.rerun.io](https://app.rerun.io/pr/5653/index.html?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG
* [x] If applicable, add a new check to the [release checklist](https://github.com/rerun-io/rerun/blob/main/tests/python/release_checklist)!

- [PR Build Summary](https://build.rerun.io/pr/5653)
- [Docs preview](https://rerun.io/preview/fa36277713991adad32d6dbfc294a02d2c1798b4/docs) 
- [Examples preview](https://rerun.io/preview/fa36277713991adad32d6dbfc294a02d2c1798b4/examples) 
- [Recent benchmark results](https://build.rerun.io/graphs/crates.html)
- [Wasm size tracking](https://build.rerun.io/graphs/sizes.html)

### CODE REVIEW

File: examples/python/llm_embedding_ner/README.md [hunk 1]
```diff
@@ -13,15 +13,92 @@ thumbnail_dimensions = [480, 480]
...
 
-This example visualizes [BERT-based named entity recognition (NER)](https://huggingface.co/dslim/bert-base-NER). It works by splitting text into tokens, feeding the token sequence into a large language model (BERT) to retrieve embeddings per token. The embeddings are then classified.
+Visualize the [BERT-based named entity recognition (NER)](https://huggingface.co/dslim/bert-base-NER) with UMAP Embeddings. 
+
+# Used Rerun Types
+[`TextDocument`](https://www.rerun.io/docs/reference/types/archetypes/text_document), [`AnnotationContext`](https://www.rerun.io/docs/reference/types/archetypes/annotation_context), [`Points3D`](https://www.rerun.io/docs/reference/types/archetypes/points3d)
+
+# Background
+It works by splitting text into tokens, feeding the token sequence into a large language model (BERT) to retrieve embeddings per token. 
+The embeddings are then classified into four types of entities: location (LOC), organizations (ORG), person (PER) and Miscellaneous (MISC).
```
Comment (nikolausWest) [inline examples/python/llm_embedding_ner/README.md:23]:
```suggestion
This example splits text into tokens, feeds the token sequence into a large language model (BERT), which outputs an embedding per token. 
The embeddings are then classified into four types of entities: location (LOC), organizations (ORG), person (PER) and Miscellaneous (MISC). The embeddings are projected to a 3D space using UMAP[https://umap-learn.readthedocs.io/en/latest/], and visualized together with all other data in Rerun.
```

File: examples/python/nuscenes/README.md [hunk 2]
```diff
@@ -16,12 +16,103 @@ build_args = ["--seconds=5"]
   <source media="(max-width: 1200px)" srcset="https://static.rerun.io/nuscenes/64a50a9d67cbb69ae872551989ee807b195f6b5d/1200w.png">
 </picture>
 
-This example visualizes the [nuScenes dataset](https://www.nuscenes.org/) using Rerun. The dataset
-contains lidar data, radar data, color images, and labeled bounding boxes.
+Visualize the [nuScenes dataset](https://www.nuscenes.org/) including lidar, radar, images, and bounding boxes data.
 
+## Used Rerun Types
+[`Transform3D`](https://www.rerun.io/docs/reference/types/archetypes/transform3d), [`Points3D`](https://www.rerun.io/docs/reference/types/archetypes/points3d), [`Boxes3D`](https://www.rerun.io/docs/reference/types/archetypes/boxes3d), [`Pinhole`](https://www.rerun.io/docs/reference/types/archetypes/pinhole), [`Image`](https://ref.rerun.io/docs/python/0.14.1/common/image_helpers/#rerun.ImageEncoded)<sup>*</sup>
+
```
Comment (nikolausWest) [inline examples/python/nuscenes/README.md:23]: Missing a background section

File: examples/python/objectron/README.md [hunk 3]
```diff
@@ -16,11 +16,101 @@ build_args = ["--frames=150"]
...
 </picture>
 
-Example of using the Rerun SDK to log the [Objectron](https://github.com/google-research-datasets/Objectron) dataset.
 
-> The Objectron dataset is a collection of short, object-centric video clips, which are accompanied by AR session metadata that includes camera poses, sparse point-clouds and characterization of the planar surfaces in the surrounding environment.
+Visualize the [Google Research Objectron](https://github.com/google-research-datasets/Objectron) dataset, which contains camera poses, sparse point-clouds and characterization of the planar surfaces in the surrounding environment.
 
+## Used Rerun Types
+ [`Points3D`](https://www.rerun.io/docs/reference/types/archetypes/points3d), [`Boxes3D`](https://www.rerun.io/docs/reference/types/archetypes/boxes3d), [`Image`](https://ref.rerun.io/docs/python/0.14.1/common/image_helpers/#rerun.ImageEncoded)<sup>*</sup>, [`Transform3D`](https://www.rerun.io/docs/reference/types/archetypes/transform3d), [`Pinhole`](https://www.rerun.io/docs/reference/types/archetypes/pinhole)
+
```
Comment (nikolausWest) [inline examples/python/objectron/README.md:24]: Missing background section

File: examples/python/open_photogrammetry_format/README.md [hunk 4]
```diff
@@ -19,10 +19,80 @@ build_args = ["--jpeg-quality=50"]
 
 Uses [`pyopf`](https://github.com/Pix4D/pyopf) to load and display a photogrammetrically reconstructed 3D point cloud in the [Open Photogrammetry Format (OPF)](https://www.pix4d.com/open-photogrammetry-format/).
 
+## Used Rerun Types
+[`Image`](https://www.rerun.io/docs/reference/types/archetypes/image), [`Points3D`](https://www.rerun.io/docs/reference/types/archetypes/points3d), [`Transform3D`](https://www.rerun.io/docs/reference/types/archetypes/transform3d), [`Pinhole`](https://www.rerun.io/docs/reference/types/archetypes/pinhole)
 
+
```
Comment (nikolausWest) [inline examples/python/open_photogrammetry_format/README.md:25]: Missing background section

**Review by nikolausWest — DISMISSED** [review]
Needs some background sections and we should also add the blueprint code sections where applicable. @jleibs, do you think we should add those directly in this PR or as a separate PR?

**Comment by andreasnaoum** [comment]
Maybe another PR for blueprints? If we wait for the blueprints, the readmes will be delayed from appearing on the website.

**Comment by Wumpf** [comment]
> should also add the blueprint code sections where applicable

let's do this in a separate pass. That's a problem with all updated readmes right now

**Comment by Wumpf** [comment]
@andreasnaoum all the example blueprint updates landed already, so whatever we want to write about blueprint in the readme can be done already now (but as I wrote before, let's do that in a separate pr)

**Review by Wumpf — APPROVED** [review]
didn't do a full review, but previous concerns seem to be all addressed, so let's move on

**Comment by Wumpf** [comment]
I'm mopping up the remaining ci failures

**Pull request merged**